### PR TITLE
one-time script to extract subset of EEBO-TCP xml files and MARC records

### PR DESCRIPTION
**Pull request by rlskoeser**
Adding to our scripts directory for documentation / replicability if needed.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -22,3 +22,12 @@ with columns corresponding to htids and their extracted version timestamps.
...
     file `version-labels/version-labels-[current date]-[i].tsv` where `i`
     is the smallest possible, postive integer.
+
+
+## EEBO-TCP subset
+
+- `get_eebo_files.py`
+
+One-time script to extracts a subset of EEBO-TCP P4 XML and MARC records
+for import into the PPA application. Setup and expected files are are documented
```
Comment (laurejt) [inline scripts/README.md:32]:
```suggestion
for import into the PPA application. Setup and expected files are documented
```

**Comment by laurejt** [comment]
@rlskoeser, can you add some additional information on the inputs / arguments for the script? It'd be nice to have the basics in the ReadMe, and it wasn't immediately clear to me what the three inputs corresponded to from description at the top of the script itself.

**Comment by rlskoeser** [comment]
> @rlskoeser, can you add some additional information on the inputs / arguments for the script? It'd be nice to have the basics in the ReadMe, and it wasn't immediately clear to me what the three inputs corresponded to from description at the top of the script itself.

Sure, I will do that. I wasn't sure how to split readme documentation from script, all the setup seemed like too much detail for the readme and better to keep with the code. Thanks for reviewing

File: scripts/README.md [hunk 2]
```diff
@@ -22,3 +22,12 @@ with columns corresponding to htids and their extracted version timestamps.
     already exists, then the output file corresponds to a new (non-existing)
     file `version-labels/version-labels-[current date]-[i].tsv` where `i`
     is the smallest possible, postive integer.
+
+
+## EEBO-TCP subset
+
+- `get_eebo_files.py`
+
+One-time script to extracts a subset of EEBO-TCP P4 XML and MARC records
```
Comment (laurejt) [inline scripts/README.md:31]:
```suggestion
One-time script that extracts a subset of EEBO-TCP P4 XML and MARC records
```

**Review by laurejt — APPROVED** [review]

**Pull request merged**